bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -581,111 +483,69 @@ All these channels except welcomePersonAtHome are read only.
 // Bridge configuration:
 Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecret="**********", username = "mail@example.com", password = "******", readStation=true, readThermostat=false] {
     // Thing configuration:
```

Comment:
This should be deleted - mixed old and new bridge configuration.

Reply:
Ok, taken in account. Corrected.